docs/ref/models/expressions.txt

```diff
@@ -314,7 +314,9 @@ The ``Func`` API is as follows:
 
     .. method:: as_sql(compiler, connection, function=None, template=None, arg_joiner=None, **extra_context)
 
-        Generates the SQL for the database function.
+        Generates the SQL fragment for the database function. Returns a tuple
+        ``(sql, params)``, where ``sql`` is the SQL string, and ``params`` is
+        the list or tuple of parameters.
```

Comment:
Maybe:
```
the list or tuple of parameters values for the SQL.
```

Reply:
Yes it's good to be clear what kind of parameters they are. "query parameters" is used throughout this file so I've changed both here to use that phrasing.